test/util/ordered_hierarchy_test.clj

```diff
@@ -88,3 +84,38 @@
     (is (= ::boolean-or-int (ordered-hierarchy/first-common-ancestor h ::boolean-or-int ::boolean-or-int)))
     (is (= ::boolean (ordered-hierarchy/first-common-ancestor h ::boolean-or-int ::boolean)))
     (is (= ::varchar-255 (ordered-hierarchy/first-common-ancestor h ::boolean ::int)))))
+
+
```

Comment:
This epic quadruple line break signifies where I'm going break this namespace in half once all the other CSV related PRs have been merged ➗